content/blog/2023/09/audit-scanner-feature.md

```diff
@@ -0,0 +1,139 @@
+`policy` and `synthetic_admission_request` to evaluate.
+
+Notice how the `synthetic_policy_requests` don't hit the Kubernetes API server;
+there's no need, as no resource is actually being CREATEd, UPDATEd, or DELETEd.
+Instead, the `synthetic_policy_request` is consumed by the PolicyServer. But
+not on its usual `validate/` HTTP endpoint, but in a different `audit/`
+endpoint, exclusively for audit requests. This `audit/` endpoint evaluates the
+policy in exactly the same way as the `validate/` one, and is instrumented as
+well, which provides separation of concerns from the normal validation of
+policies.
+
+Once the Audit Scanner has the audited results, it writes its findings using
+two new types of Kubernetes Custom Resources: `PolicyReport` and
+`ClusterPolicyReport`, [from the K8s Policy Working Group](https://github.com/kubernetes-sigs/wg-policy-prototypes/tree/master/policy-report).
+
+Each Kubernetes Namespace inspected by the Audit Scanner will feature one
+`PolicyReport` object that contains the results of the audit. For cluster-wide
+resources (like Namespace, PersistentVolume, &hellip;) the Audit Scanner
+creates one `ClusterPolicyReport` object per cluster.
+
+There are some resources that the Audit Scanner cannot audit in this first
+implementation. For example, it cannot simulate UPDATE requests, as it doesn't know
+exactly which part of the resource needs to be changed in a meaningful way.
+Also, the DELETE requests will be synthesized in the future. Read the
+[limitations](https://docs.kubewarden.io/explanations/audit-scanner/limitations)
+doc for an expanded explanation. The full [explanation docs
+page](https://docs.kubewarden.io/explanations/audit-scanner) has further explanation.
+
+The new Audit Scanner brings all this functionality while keeping in line with the
+modular architecture of Kubewarden. It allows operators to scale it as needed,
+configure its periodicity, monitoring and tracing, and making sure it doesn't
+influence the PolicyServers in a bad way.
+
+#### Fine, but what about a UI?
+
+Reading `(Cluster)PolicyReports` sounds cumbersome?
+
+Don't worry, the community has created the nice [Policy Reporter UI
+project](https://kyverno.github.io/policy-reporter), which provides a view on
+the reports and ways to filter them. `Screenshot >= 1000*word`:
+
+{{<figure src="/images/audit-scanner/policy-reporter-ui.png" alt="policy-reporter-ui view"  >}}
+
+We have elected to include the `policy-reporter-ui` chart in our
+`kubewarden-controller` chart as a subchart. It is disabled by default, have a
+look at our [Audit Scanner install
+docs](https://docs.kubewarden.io/howtos/audit-scanner) to see how to enable it.
+
+Policy Reporter UI is not the only tool that can consume results from the
+PolicyReports. More and more tools are consuming PolicyReports themselves,
+which is great for the community and the policy engines.
+
+## Do I need to change anything?
+
+No :).
```

Comment:
Just to ensure proper rendering

```suggestion
No 🙂.
```

Reply:
https://github.com/kubewarden/kubewarden.io/pull/194/commits/87d198e6578733ee8605fff80cc8904f655c5ec0